README.md

```diff
@@ -20,10 +20,20 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   cd ..
   catkin_make
   ```
+  
+### Application can work at two modes:
+-  Local mode
+  -  All the ROS nodes run at the same machine.
+-  Network mode
+  -  ROS nodes run at two different machine: Server and Client.
+  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
+  -  Server runs all other ROS nodes except two nodes running on client.
```

Comment:
what I'm trying to say that the words "simulated robot" is little bit confusing for me.  rviz can be thought as simulated robot(old Readme use this way), and RobotStudio can also be called simulated robot. is there better way to distinguish these two? 

If I understand you right here, basically you advice me to put more general information here. How about we write like this.
> 
> ### Application can work at two modes:
> -  Local mode
>   -  All the functions run at the same machine.
> -  Network mode
>   -  Functions are distributed to different machine: Server and Client.
>   -  Client runs ROS nodes rviz to control the hardware(robot and sensor), simulated robot and simulated sensor can also be used.
>   -  Server runs other function to process point cloud and trajectory data.
>   Note: One server can support multiple clients.